### ISSUE TITLE
Shaders compilation fails on Intel drivers (missing 'GL_ARB_explicit_uniform_location' extension)

### ISSUE THREAD

**Issue by kemenaran** [citra-emu/citra#1222]
On systems with Intel integrated graphics, shaders compilation fails with several of these errors:

```
Render.OpenGL <Error> video_core/renderer_opengl/gl_shader_util.cpp:LoadProgram:58:
Error compiling fragment shader:
extension 'GL_ARB_explicit_uniform_location' is not supported

Render.OpenGL <Error> video_core/renderer_opengl/gl_shader_util.cpp:LoadProgram:81:
Error linking shader:
ERROR: One or more attached shaders not successfully compiled
```

This error appeared after the merge of the [GSLS Fragment Shaders Generation PR](https://github.com/citra-emu/citra/pull/1187). Reproduced by @SeannyM on Intel HD 4000 on both Windows with the latest drivers (15.33.39.4276) and OS X.

Results in almost nothing being rendered:

<img width="512" alt="8c78abd6-7b07-11e5-94a4-d426b9b893bf" src="https://cloud.githubusercontent.com/assets/179923/10723223/f34f7838-7bb8-11e5-9a29-5a7b851c1113.png">

/cc @yuriks

**Comment by Subv** [comment]
It works correctly on Intel HD 4600

**Comment by Subv** [comment]
Fixed by #1253
